rotkehlchen/db/dbhandler.py

```diff
@@ -338,6 +341,8 @@ def _run_actions_after_first_connection(self, password: str) -> None:
             )
             self.conn_transient.commit()
 
+        db_schema_sanity_check(cursor=self.conn.cursor(), db_name='encrypted')
```

Comment:
here you are creating a cursor, the old way, not passing it through the proper context manager and never closing it.